Lib/unittest/mock.py

```diff
@@ -1062,6 +1062,10 @@ def _calls_repr(self, prefix="Calls"):
         return f"\n{prefix}: {safe_repr(self.mock_calls)}."
 
 
+# gh-100690 Denylist for forbidden attribute names in safe mode
```

Comment:
Personally I dislike these kind of bug links, the comment is fine without it, and in a decade's time when we're no longer using GitHub for issue tracking, this will be a wart ;-)

Reply:
Same for me